curp/src/server/raw_curp/tests.rs

```diff
@@ -670,3 +677,105 @@ fn is_synced_should_return_true_when_followers_caught_up_with_leader() {
     curp.lst.update_match_index(s2_id, 3);
     assert!(curp.is_synced());
 }
+
+#[traced_test]
+#[test]
+fn add_node() {
```

Comment:
Please add your expectations in the test name like other tests do.

Reply:
Fixed